php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+					}
+					chargebeeCustomerDetail = await createChargebeeCustomer(newCbCustomer);
+					if(chargebeeCustomerDetail?.customer) {
+						CB_CUSTOMER_ID = chargebeeCustomerDetail.customer.id
+					}
+				}else{
+					/** @condition for existing Customer
+					 /* chargebeeCustomerDetail = { Your existing customer detail };
+					 /* CB_CUSTOMER_ID = chargebeeCustomerDetail.customer.id
+					 */
+				}
+				console.info(`=============== Customer Created ================`);
+				console.info(chargebeeCustomerDetail);
+
+				console.info(`=============== Payment Initated ================`);
+				let estimationDetails = await createCbEstimate();
+				let estimationAmount;
+				let currencyCode;
+				if (estimationDetails?.estimate?.invoice_estimate) {
+					estimationAmount = estimationDetails.estimate.invoice_estimate.sub_total;
+					currencyCode = estimationDetails.estimate.invoice_estimate.currency_code;
+					console.info(`=======Chargebee's Estimation : ${estimationAmount}==========`);
+				}
+				/**
+				 * @note this will create a new customer in razorpay app
+				 */
+				let newRazorpayCustomer = await createRazorpayCustomer({
+					"name": `${chargebeeCustomerDetail.first_name || ''} ${chargebeeCustomerDetail.last_name || ''}`,
+					"email": chargebeeCustomerDetail.email || null
+				});
+				/**
+				 * @note For existing razorpay customers, skip the above part and pass the respective customer details
+				 */
+				console.info(
+					`======== Customer Details ==========`
+				);
+				console.info(newRazorpayCustomer);
+				let orderDetails;
+				if (newRazorpayCustomer.id) {
+					orderDetails = await createOrder(newRazorpayCustomer.id, estimationAmount, currencyCode);
+					console.info(
+						`=========== Order Created ===========`
+					);
+					console.info(orderDetails);
+					if (orderDetails && orderDetails.id) {
+						console.info(
+							`========== Inintating Razorpay Checkout ==========`
+						);
+						openRPCheckout(
+								newRazorpayCustomer,
+								orderDetails.id,
+								orderDetails.amount,
+								orderDetails.currency
+						);
+					}
+				}
+				console.info(
+				"========= Payment Done =========== "
+				);
+			} catch (err) {
+				payButton.classList.remove('disable')
+				console.log("Error:", err);
+			}
+		}
+
+		async function createChargebeeCustomer(customerDetail) {
+			console.info(
+					"====== Creating CB Customer =========="
+			);
+			let myHeaders = new Headers();
+			myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+			let urlencoded = new URLSearchParams();
+			urlencoded.append("first_name", customerDetail.first_name || '');
+			urlencoded.append("last_name", customerDetail.last_name || '');
+			urlencoded.append("email", customerDetail.email || '');
+
+			let requestOptions = {
+				method: 'POST',
+				headers: myHeaders,
+				body: urlencoded,
+			};
+
+			let newCbCustomer = await fetch("http://localhost:3000/api/create_cb_customer", requestOptions)
```

Comment:
Extract the server URL out to a global variable, then build the URLs while making the request